config/ingest.go

```diff
@@ -134,6 +138,10 @@ func (c KafkaConfiguration) CreateKafkaConfig() kafka.ConfigMap {
 		config["metadata.max.age.ms"] = 3 * c.TopicMetadataRefreshInterval
 	}
 
+	if c.DebugContexts != nil {
+		config["debug"] = strings.Join(c.DebugContexts, ",")
+	}
```

Comment:
Empty list may result in an empty string. Might be better to check for length?

Reply:
Valid. Will fix it promptly.

Reply:
Fixed.